worldengine/generation.py

```diff
@@ -105,12 +105,20 @@ def initialize_ocean_and_thresholds(world, ocean_level=1.0):
     """
     e = world.elevation['data']
     ocean = fill_ocean(e, ocean_level)
-    hl = find_threshold_f(e, 0.10)  # the highest 10% of all (!) land are declared hills
-    ml = find_threshold_f(e, 0.03)  # the highest 3% are declared mountains
+
+    # values chosen to emulate the exponential behavior seen here:
+    # http://www.ngdc.noaa.gov/mgg/global/etopo1_surface_histogram.html
+    pl = find_threshold_f(e, 0.95, ocean=ocean)  # the highest x% of land are declared plains; save some for beaches (?)
+    hl = find_threshold_f(e, 0.50, ocean=ocean)  # the highest x% are declared hills
```

Comment:
The first two plots in https://github.com/Mindwerks/worldengine/pull/203 show the difference of the values pretty nicely (old levels first, then new levels). The change is not as drastic as it might seem.